REFERENCE.md

```diff
@@ -142,7 +178,7 @@ Data type: `Optional[String]`
 
 The path to an .erb or .epp template file or `undef`.
 
-### extlib::dir_split
+### `extlib::dir_split`
```

Comment:
Perhaps we should first regen the README in a separate PR/commit

Reply:
done in https://github.com/voxpupuli/puppet-extlib/pull/169